From 24bac122bf4ffe437bb4f6ca633ad08f31a23d73 Mon Sep 17 00:00:00 2001
From: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
Date: Mon, 29 Dec 2014 13:56:54 +0100
Subject: [PATCH 5/5] Add stack direction for the Nios II architecture

According to Altera Application Binary Interface document for the Nios
II architecture
(http://www.altera.com/literature/hb/nios2/n2cpu_nii51016.pdf):

 "The stack grows downward (i.e. towards lower addresses). ".

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
---
 m4/stack-direction.m4 | 1 +
 1 file changed, 1 insertion(+)

diff --git a/m4/stack-direction.m4 b/m4/stack-direction.m4
index 1a97d5a..d9c5157 100644
--- a/m4/stack-direction.m4
+++ b/m4/stack-direction.m4
@@ -38,6 +38,7 @@ AC_DEFUN([SV_STACK_DIRECTION],
       mmix | \
       mn10200 | \
       mn10300 | \
+      nios2 | \
       ns32k | \
       pdp11 | \
       pj* | \
-- 
2.1.0

